Makefile

```
@@ -376,7 +389,9 @@ ifeq ($(UNAME_S), Darwin)
 	# Fix the rpath for the dylib
 	install_name_tool -id "@rpath/libwasmer.dylib" package/lib/libwasmer.dylib
 else
-	cp target/release/libwasmer_c_api.so package/lib/libwasmer.so
+	if [ -f "target/$(HOST_TARGET)/release/wasmer-headless" ]; then \
```

I presume you wanted to write:

```suggestion
	if [ -f "target/$(HOST_TARGET)/wasmer-headless" ]; then \
```

But why moving `libwasmer_c_api.so` to `libwasmer.so` if `wasmer-headless` exists?

---

I'm also unclear on the logic here